docs/src/pages/guides/DB2OnCloudSetup.md

```diff
@@ -2,146 +2,173 @@
+        1. Enter the Name. Any name works for example, `ibm-fhir-server-db2`.
+        1. Click Add
 
-These properties are needed to deploy and manage the IBM FHIR Server schema.
+1. Access the credential, select View Credentials. The entry you just created looks like the following block of JSON(the secrets are blanked out here):
 
+    ``` json
+    {
+        "hostname": "dashdb-txn-flex-.***********.services.dal.bluemix.net",
+        "password": "bluadmin-password-very-secret",
+        "https_url": "https://dashdb-txn-flex-************.services.dal.bluemix.net:8443",
+        "port": 50000,
+        "ssldsn": "DATABASE=BLUDB;HOSTNAME=dashdb-txn-flex-************.services.dal.bluemix.net;PORT=50001;PROTOCOL=TCPIP;UID=bluadmin;PWD=bluadmin-password-very-secret;Security=SSL;",
+        "host": "dashdb-txn-flex-************.services.dal.bluemix.net",
+        "jdbcurl": "jdbc:db2://dashdb-txn-flex-************.services.dal.bluemix.net:50000/BLUDB",
+        "uri": "db2://bluadmin:bluadmin-password-very-secret@dashdb-txn-flex-************.services.dal.bluemix.net:50000/BLUDB",
+        "db": "BLUDB",
+        "dsn": "DATABASE=BLUDB;HOSTNAME=dashdb-txn-flex-************.services.dal.bluemix.net;PORT=50000;PROTOCOL=TCPIP;UID=bluadmin;PWD=bluadmin-password-very-secret;",
+        "username": "bluadmin",
+        "ssljdbcurl": "jdbc:db2://dashdb-txn-flex-************.services.dal.bluemix.net:50001/BLUDB:sslConnection=true;"
+    }
+    ```
 
-### FHIRSERVER User and API Key
+4. Save these details for later, as these properties are needed to deploy and manage the IBM FHIR Server schema.
 
-The **BLUADMIN** user is used to deploy the initial schema objects (tables, indexes, stored procedures etc). Following the least-privilege principle, the FHIR server itself does not use **BLUADMIN**. The FHIR server uses an API Key associated with an IAM Service Id. This Service Id is mapped to a Db2 user which is granted explicit privileges to the tables and stored procedures.
+This **BLUADMIN** user is used to deploy the schema objects (tables, indexes, stored procedures etc) only. 
 
-```
-    API KEY -------> Service Id -------> DB2 User ------> SELECT/UPDATE/EXECUTE etc Privileges
- (FHIR Config)          (IAM)              (DB2)                          (DB2)
-```
+### **Add the FHIRSERVER User and API Key**
 
-On the IBM Cloud console, select Manage / Access (IAM).
+Following the least-privilege principle, the IBM FHIR server itself does not use **BLUADMIN**. The IBM FHIR server uses an API Key associated with an [IAM Service Id](https://cloud.ibm.com/iam/serviceids). The IAM Service Id is mapped to a Db2 user which is granted explicit privileges to the tables and stored procedures.
 
-Select the [Service IDs panel](https://cloud.ibm.com/iam/serviceids).
+The IBM FHIR Server uses the access flow: 
 
-Wait for the table to populate. If you do not yet have an appropriate service id, click `Create (+)`
+1. Read API Key from the fhir-server-config.json
```

Comment:
worth mentioning the tenantKey here as well?

Reply:
done